spec/models/html_document_spec.rb

```diff
@@ -85,18 +86,73 @@
         expect(description).to eq 'My OG Description'
       end
     end
+
+    context 'when a Dublin Core description is available' do
+      let(:raw_document) { doc_with_dc_data }
+
+      it 'returns the open graph description' do
```

Comment:
I think this should say "returns the Dublin Core description" instead of "returns the open graph description".